Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/Logging/DefaultLogMessageFormatter.cs

```diff
@@ -48,21 +50,53 @@ public override string FormatMessage(MessageState state)
                 message = ApplyMessageProperties(state.MessageTemplate, messageProperties, state.MessageArguments);
             }
 
+            var displayLevel = state.Level != null ? ConvertLogLevelToLabel(state.Level.Value) : null;
+
+            var sb = new StringBuilder(
+                                    25 + // Length for timestamp
+                                    (state.AwsRequestId?.Length).GetValueOrDefault() +
+                                    displayLevel.Length +
+                                    (message?.Length).GetValueOrDefault() +
+                                    5 // Padding for tabs
+                                    );
+
             if (!AddPrefix)
             {
                 return message;
             }
 
-            var displayLevel = state.Level != null ? ConvertLogLevelToLabel(state.Level.Value) : null;
 
-            if (!string.IsNullOrEmpty(displayLevel))
+            if (AddPrefix)
```

Comment:
Don't we always enter this given L63? If so might be simpler to remove.

Reply:
Fixed. It was the previous line 63 block that was leftover from previous version and needed to be deleted.